src/EnergyPlus/PlantComponent.hh

```diff
@@ -0,0 +1,20 @@
+#ifndef PLANTCOMPONENT_HH_INCLUDED
+#define PLANTCOMPONENT_HH_INCLUDED
+
+#include <string>
+#include <memory>
+
+class PlantComponent
+{
+public:
+	std::string name;
+	int compType;
```

Comment:
That's a great idea @kbenne looking forward.  Whenever we do talk about this we'll keep it in mind.  One thing I want to balance is how this current approach is set up to work perfectly within the existing structures while still making big strides on a component-by-component level.  If the IddObjectType can be used in this same fashion; awesome.  If not, I'm not sure we'll pull it in until we've pushed this along quite a ways.